Add automated testing like Travis CI, CircleCI, etc?

[flake8](http://flake8.pycqa.org) testing of https://github.com/trustedsec/ptf on Python 3.7.1

$ __flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics__
```
./modules/vulnerability-analysis/owtf.py:29:501: E999 SyntaxError: invalid syntax
AFTER_COMMANDS="cd {INSTALL_LOCATION}, python setup.py install, systemctl enable postgresql, /etc/init.d/postgresql start, export OWTF_USER=$(grep DATABASE_USER owtf/settings.py | tail -1 | cut -d'"' -f2), export OWTF_PASS=$(grep DATABASE_PASS owtf/settings.py | tail -1 | cut -d'"' -f2), export OWTF_DB=$(grep DATABASE_NAME owtf/settings.py | tail -1 | cut -d'"' -f2), psql -c \"CREATE USER $OWTF_USER WITH PASSWORD $OWTF_PASS\", psql -c \"CREATE DATABASE $OWTF_DB WITH OWNER $OWTF_USER ENCODING 'utf-8' TEMPLATE template0;\""
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    ^
./modules/custom_list/list.py:1:1: F821 undefined name 'burp'
burp
^
./modules/custom_list/list.py:2:1: F821 undefined name 'metasploit'
metasploit
^
./modules/custom_list/list.py:3:1: F821 undefined name 'zaproxy'
zaproxy
^
./modules/custom_list/list.py:4:1: F821 undefined name 'nmap'
nmap^
1     E999 SyntaxError: invalid syntax
4     F821 undefined name 'burp'
5
```
__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. These 5 are different from most other flake8 issues which are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

--- a/.github/workflows/Python_tests.yml
+++ b/.github/workflows/Python_tests.yml
@@ -16,7 +16,9 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install flake8  # pytest -r requirements.txt
+          pip install black flake8 pytest -r requirements.txt
+      #- name: Check formatting with black
+      #  run: black --check .
       - name: Lint with flake8
         run: |
           # stop the build if there are Python syntax errors or undefined names

--- a/.github/workflows/Python_tests.yml
+++ b/.github/workflows/Python_tests.yml
@@ -1,0 +1,29 @@
+name: Python_tests
+on: [push, pull_request]
+jobs:
+  Python_tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        python-version: [2.7, 3.6, 3.7, 3.8] # 3.5,
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8  # pytest -r requirements.txt
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+          flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+      #- name: Test with pytest
+      #  run: pytest
+      #- name: Run doctests with pytest
+      #  run: pytest --doctest-modules

